webui/react/src/services/decoder.ts

```diff
@@ -128,7 +128,10 @@ const jsonToExperimentConfig = (data: unknown): ExperimentConfig => {
   const io = decode<ioTypeExperimentConfig>(ioExperimentConfig, data);
   const config: ExperimentConfig = {
     description: io.description,
-    resources: {},
+    labels: io.labels,
+    resources: {
```

Comment:
**question:** is `resources` needed?

Reply:
it should be..  we must have removed it another PR? it's used to show the max_slots in experiment infobox.

Reply:
oh it's being added down below..